lib/puppet/provider/rds_instance/v2.rb

```diff
@@ -40,6 +42,14 @@ def self.prefetch(resources)
 
   def self.db_instance_to_hash(region, instance)
     db_subnet = instance.db_subnet_group ? instance.db_subnet_group.db_subnet_group_name : nil
+
+    # tags stuff requires aws sdk >= 2.6.11
```

Comment:
We should. This version added the arn parameter to rds resources which is vital to getting and setting tags in RDS